adopterAPI.json

```diff
@@ -420,6 +605,11 @@
             "type": "string",
             "format": "uuid",
             "nullable": true
+          },
+          "adopterId": {
+            "type": "string",
+            "format": "uuid",
+            "nullable": true
           }
```

Comment:
We don't need this. We decided to just pull this from JWT token in some previous PR.